pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -584,7 +584,7 @@ def _ensure_init_storage(self):
         # else
         print(_("Reverting previous configuration. This may take a moment..."))
         reset_storage(self.storage, scan_all=True)
-        self._manual_part_proxy.SetMountPoints([])
+        self._manual_part_proxy.SetRequests([])
```

Comment:
From this part I wouldn't understand what kind of storage requests are we talking about. Could you make this more obvious that this is mount point requests?

Reply:
Added a comment.